### PR TITLE
Documentation for nf-core 

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker` --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,19 +17,50 @@
...
+7. QC reports generation [`pmultiqc`](https://github.com/bigbio/pmultiqc)
+
+DDA-ISO:
+
+1. RAW file conversion to mzML ([`thermorawfileparser`](https://github.com/compomics/ThermoRawFileParser))
+2. Peptide identification using [`comet`](https://uwpr.github.io/Comet/) and/or [`msgf+`](https://github.com/MSGFPlus/msgfplus)
+3. Re-scoring peptide identifications [`percolator`](https://github.com/percolator/percolator)
+4. Peptide identification FDR [`openms fdr tool`](https://github.com/ypriverol/quantms/blob/dev/modules/local/openms/falsediscoveryrate/main.nf)
+5. Extracts and normalizes isobaric labeling [`IsobaricAnalyzer`](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_IsobaricAnalyzer.html)
+6. Protein inference [`ProteinInference`](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_ProteinInference.html)
```
Comment (jpfeuffer) [inline README.md:54]: or epifany

**Review by jpfeuffer — CHANGES_REQUESTED** [review]
cool! you can accept some of my changes and add a few other tools.

File: README.md (hunk 1 above)
Comment (ypriverol) [reply README.md]: Done

File: CITATIONS.md [hunk 2]
```diff
@@ -10,10 +10,26 @@
...
+    > Demichev V, Messner CB, Vernardis SI, Lilley KS, Ralser M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Nat Methods. 2020 Jan;17(1):41-44. doi: 10.1038/s41592-019-0638-x. Epub 2019 Nov 25. PMID: 31768060; PMCID: PMC6949130.
+
+* [MSstats](https://www.ncbi.nlm.nih.gov/pubmed/24794931/)
+    > Choi M., Chang CY., Clough T., Broudy D., Killeen T., MacLean B., Vitek O. (2014). MSstats: an R package for statistical analysis of quantitative mass spectrometry-based proteomic experiments. Bioinformatics (Oxford, England), 30(17), 2524–2526. doi: 10.1093/bioinformatics/btu305. PubMed PMID: 24794931.
+
+* [Comet](https://www.ncbi.nlm.nih.gov/pubmed/23148064/)
+    > Eng JK., Jahan TA., Hoopmann MR. (2013). Comet: an open-source MS/MS sequence database search tool. Proteomics, 13(1), 22–24. doi: 10.1002/pmic.201200439. PubMed PMID: 23148064
+
+* [MS-GF+](https://www.ncbi.nlm.nih.gov/pubmed/25358478/)
+    > Kim S., Pevzner PA. (2014). MS-GF+ makes progress towards a universal database search tool for proteomics. Nature communications, 5, 5277. doi: 10.1038/ncomms6277. PubMed PMID: 25358478; PubMed Central PMCID: PMC5036525
```
[the remark this answers is not in the document]
Comment (ypriverol) [reply CITATIONS.md:32]: Done

File: README.md [hunk 3]
```diff
@@ -17,19 +17,50 @@
...
 <!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
 
-1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
-2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+The quantms allows uses to perform analysis in three main type of analytical MS-based quantitative methods: DDA-LFQ, DAA-ISO, DIA-LFQ. Each of these workflows share some processes but also includes their own steps. In summary:
+
+DDA-LFQ:
+
+1. RAW file conversion to mzML ([`thermorawfileparser`](https://github.com/compomics/ThermoRawFileParser))
+2. Peptide identification using [`comet`](https://uwpr.github.io/Comet/) and/or [`msgf+`](https://github.com/MSGFPlus/msgfplus)
```
[the remark this answers is not in the document]
Comment (ypriverol) [reply README.md:40]: Done

File: README.md [hunk 4]
```diff
@@ -17,19 +17,50 @@
...
+3. Re-scoring peptide identifications [`percolator`](https://github.com/percolator/percolator)
+4. Peptide identification FDR [`openms fdr tool`](https://github.com/ypriverol/quantms/blob/dev/modules/local/openms/falsediscoveryrate/main.nf)
+5. Quantification: Feature detection [`proteomicsLFQ`](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/UTILS_ProteomicsLFQ.html)
+6. Protein inference and quantification [`proteomicsLFQ`](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/UTILS_ProteomicsLFQ.html)
+7. QC reports generation [`pmultiqc`](https://github.com/bigbio/pmultiqc)
+
+DDA-ISO:
+
+1. RAW file conversion to mzML ([`thermorawfileparser`](https://github.com/compomics/ThermoRawFileParser))
+2. Peptide identification using [`comet`](https://uwpr.github.io/Comet/) and/or [`msgf+`](https://github.com/MSGFPlus/msgfplus)
```
[the remark this answers is not in the document]
Comment (ypriverol) [reply README.md:52]: Done

File: docs/output.md [hunk 5]
```diff
@@ -8,52 +8,66 @@ The directories listed below will be created in the results directory after the
...
+    - [searchenginecomet/*.idXML](#identifications)
+    - [searchenginemsgf/*.idXML](#identifications)
+- consensusID identifications:
+    - [consensusid/*.idXML](#identifications)
+- pipeline information:
+    - [pipeline_info/...](#nextflow-pipeline-info)
+- DDA-LFQ quantification results:
+    - [proteomicslfq/out.consensusXML](#consenusxml)
+    - [proteomicslfq/out_msstats.csv](#msstats-ready-quantity-table)
+    - [proteomicslfq/out_triqler.tsv](#triqler)
```
Comment (jpfeuffer) [inline docs/output.md:62]: link probably missing

Comment (jpfeuffer) [reply docs/output.md:62]: it needs to be a header somewhere.

Comment (ypriverol) [reply docs/output.md:62]: Sorry, i forgot it, Now is included.

**Pull request merged**